tooling/bundler/src/bundle/windows/sign.rs

```diff
@@ -17,14 +19,164 @@ use winreg::{
   RegKey,
 };
 
-pub struct SignParams {
+/// Enum to hold the different signing params depending on the current setup. (Signtool or AzureSignTool)
+pub enum SignParams {
+  SignTool(SignToolParams),
+  Azure(AzureSignToolParams),
+}
+
+impl SignParams {
+  /// Locates the signing tool executable
+  fn locate_tool(&self) -> crate::Result<PathBuf> {
```

Comment:
Seems like is never used, let's remove for now

Reply:
removed!